test/functionalTest/cases/3122_location_metadata_issues/create_location_attr_V2.test

```diff
@@ -0,0 +1,166 @@
+# Copyright 2018 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
This test covers the following cases (both in the left branch but with different results in the checkings)

![imagen](https://user-images.githubusercontent.com/1534240/48832033-80596200-ed78-11e8-90c7-a5094581fb89.png)

Great!

NTC

Reply:
It's ok